robot/src/main/java/us/ilite/robot/modules/EElevatorState.java

```diff
@@ -0,0 +1,23 @@
+package us.ilite.robot.modules;
+
+public enum EElevatorState {
+
+    //TODO find all of the values for the power.
+    NORMAL(0),
+    STOP(0),
+    HOLD(0),
```

Comment:
There shouldn't need to be a hold position - the PID loop + gravity feedforward should be what holds the elevator in position